docs/admin/troubleshooting.md

```diff
@@ -414,6 +414,138 @@ Verify if you're using Infiniband for `fabric_iface`: in the server config. The
 		$ dmg pool destroy --pool=$DAOS_POOL --force
 		Pool-destroy command succeeded
 
+## Diagnostic and Recovery Tools
+
+### **WARNING : Please be careful and use this tool under supervision of DAOS export.**
```

Comment:
Pls don't add warning as subsection and use !!! warning instead.

Reply:
Done